model/signature_handler.go

```diff
@@ -47,6 +48,8 @@ type SignatureValidationResult struct {
 	// TODO(gunnsth): Add more fields such as ability to access the certificate information (name, CN, etc).
 	// TODO: Also add flags to indicate whether the signature covers the entire file, or the entire portion of
 	// a revision (if incremental updates used).
+
+	GeneralizedTime time.Time
```

Comment:
Add a comment that describes what this is?  Only applicable for document timestamp?